mautrix/client/api/events.py

```diff
@@ -16,6 +16,10 @@
 from mautrix.types.event.state import state_event_content_map
 from .base import BaseClientAPI
 
+API_CALLS = Counter("bridge_matrix_api_calls",
+                    "The number of Matrix client API calls made", ("update_type",))
+API_CALLS_FAILED = Counter("bridge_matrix_api_calls_failed",
+                    "The number of Matrix client API calls which failed", ("update_type",))
```

Comment:
Do these need to be imported in a web endpoint function or does a creation of `Counter()` already add the metric to a Prometheus endpoint?

Reply:
It's magic, you don't need to do anything :)